practice-two/src/styles/components/input.css

```diff
@@ -0,0 +1,6 @@
+.input {
```

Comment:
Update naming convention. You can refer Bootstrap to define class names for input